std/meta.d

```diff
@@ -857,6 +857,13 @@ template staticMap(alias F, T...)
 
     else
     {
+        /* While:
+         *   alias staticMap = aliasSeq!(F!T[0], staticMap!(F, T[0 .. $]));
```

Comment:
Even though it's an uncompiled comment, The second parameter should be `T[1 .. $]`

Reply:
And `AliasSeq` should be capitalized.